srcpkgs/fake-hwclock/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'fake-hwclock'
+pkgname=fake-hwclock
+version=0.11
+revision=1
+archs=noarch
+create_wrksrc=yes
+short_desc="Save/restore system clock on machines without working RTC hardware"
+maintainer="Ivan Gonzalez Polanco <ivan14polanco@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://tracker.debian.org/pkg/fake-hwclock"
+distfiles="${DEBIAN_SITE}/main/f/${pkgname}/${pkgname}_${version}.tar.gz"
+checksum="58e29ff272a8e8b40ab972c49f82fd8fb6ef69a7fdde5f84292f800b53ea29ce"
+
+do_extract() {
+	filename="$(basename ${distfiles})"
+	tar xzf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${filename} --strip-components=1 -C ${wrksrc}
+}
+
+do_install() {
+	vbin ${pkgname}
+	vconf etc/default/${pkgname}
```

Comment:
set `conf_files=etc/default/fake-hwclock` in top level

Reply:
I set said variable but I'm getting an error:
```
ERROR: fake-hwclock-0.11_1: '/etc/default/fake-hwclock' configuration file not in PKGDESTDIR!
```
I tried moving the values around (the documentation says that the path should be absolute) but I'm not sure what I'm doing wrong.

Reply:
Maybe I should make it like this?
https://github.com/void-linux/void-packages/blob/da4ff0af1cc2a423667335f6b81b60caead79181/srcpkgs/boinc/template#L100-L102

Reply:
Do both `vsconf` to put file into xbps package and `conf_files=` to instruct xbps-install to not overwrite it on update.

Reply:
The config file is present in `$PKGDESTDIR/etc/`, but I'm not sure how to move it into `$PKGDESTDIR/etc/default/`

Reply:
With:
```diff
-vconf etc/default/fake-hwclock
+vinstall etc/default/fake-hwclock 644 etc/default
```
It works as intended, is that alright?
I think the error was due to `vconf` moving `<file>` directly beneath /etc, without the option to move it to a subdirectory of /etc.